pom.xml

```diff
@@ -21,11 +21,15 @@
   </licenses>
   <properties>
     <powermock.version>1.6.2</powermock.version>
-    <jenkins.version>1.580.1</jenkins.version>
```

Comment:
By removing this, what version of Jenkins becomes the version it is built against? You can't pick anything newer than the latest LTS and MANY people have not moved to 2.x yet, so that is out. This plugin is used by too many people to make a breaking change.